### PR TITLE
Ports balloon alerts

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About the Pull Request

Ports balloon alerts from TG.

## Why It's Good For The Game

Good UI + makes it easier to port stuff in the future

## Changelog

:cl:
code: Balloon alerts ported from TG.
/:cl:

### CODE REVIEW

**Comment by 77seven77seven** [comment]
how do these even look

**Comment by Vizoplez** [comment]
The one guy spamming 'HELP ME IM BEING SEXUALLY ASSAULTED' crashing the MC:

Put simply, you might want to add an anti-flood mechanism.

File: code/__defines/_planes+layers.dm [hunk 1]
```diff
@@ -191,21 +191,23 @@ What is the naming convention for planes or layers?
 	#define BLIND_LAYER         3
 	#define CRIT_LAYER          4
 
-#define VISION_CONE_PLANE               7
+#define VISION_CONE_PLANE				7
```
Comment (Tsurupeta) [inline code/__defines/_planes+layers.dm]: dude, spaces are there for a reason, either bring them back or remove alltogether

**Review by Tsurupeta — APPROVED** [review]
I assume it's fine

**Pull request merged**